python/lib/evaluations/dmod/evaluations/specification/README.md

```diff
@@ -0,0 +1,1338 @@
+and what thresholds to use.
+
+<a id="ThresholdSpecificationExamples"></a>
+### Example
+
+The following two examples load an RDB file named `resources/nwis_stat_thresholds.rdb`,
+names locations based off of the identified `site_no` column, matches loaded thresholds onto two new columns, the first
+being `threshold_day` on the threshold data, created by converting the `month_nu` and `day_nu` fields into `Day` objects,
+the second being named `threshold_day` on the observation data, created by converting the `value_date` field into
+`Day` objects. Use the `p75_va`, `p80_va`, and `p50_va` fields as thresholds named `"75th Percentile"`,
+`"80th Percentile"`, and `"Median"`, respectively, measured in `ft^3/s` and weighing `10`, `5`, and `1`, respectively.
+The `75th Percentile` threshold will be considered 10 times more important than the `Median`, while the
+`80th Percentile` will be considered half as important as the `75th Percentile`, but 5 times as important as the
+`Median`.
+
+```json
+{
+    "name": "NWIS Stat Percentiles",
+    "backend": {
+        "backend_type": "file",
+        "format": "rdb",
+        "address": "resources/nwis_stat_thresholds.rdb"
+    },
+    "locations": {
+        "identify": true,
+        "from_field": "column",
+        "pattern": "site_no"
+    },
+    "application_rules": {
+        "threshold_field": {
+            "name": "threshold_day",
+            "path": [
+                "month_nu",
+                "day_nu"
+            ],
+            "datatype": "Day"
+        },
+        "observation_field": {
+            "name": "threshold_day",
+            "path": [
+                "value_date"
+            ],
+            "datatype": "Day"
+        }
+    },
+    "definitions": [
+        {
+            "name": "75th Percentile",
+            "field": "p75_va",
+            "weight": 10,
+            "unit": {
+                "value": "ft^3/s"
+            }
+        },
+        {
+            "name": "80th Percentile",
+            "field": "p80_va",
+            "weight": 5,
+            "unit": {
+                "value": "ft^3/s"
+            }
+        },
+        {
+            "name": "Median",
+            "field": "p50_va",
+            "weight": 1,
+            "unit": {
+                "value": "ft^3/s"
+            }
+        }
+    ]
+}
+```
+
+The following example can produce the exact same results as the example from above but through the use of templates.
+
+```json
+{
+    "backend": {
+        "template_name": "NWIS Stat Thresholds"
+    },
+    "locations": {
+        "template_name": "Site Number Column"
+    },
+    "application_rules": {
+        "template_name": "Date to Day"
+    },
+    "definitions": [
+        {
+            "template_name": "75th Percentile"
+        },
+        {
+            "template_name": "80th Percentile"
+        },
+        {
+            "template_name": "Median"
+        }
+    ]
+}
+```
+
+<a id="UnitDefinition"></a>
+## Unit Definition
```

Comment:
Is there a list of supported units / a dependency (i.e. `pint`) that you can point to in this section?